elements/progress-slider/progress-slider.js

```diff
@@ -55,6 +55,8 @@ let self = class ProgressSlider extends HTMLElement {
 				element ??= slot.firstElementChild;
 				this[elementProp] = element;
 
+				["min", "max", "step"].forEach(prop => this[elementProp][prop] = this[prop]);
```

Comment:
First, we should check if we actually have a new element before running any side effects. Second, the slider is always the source of truth here. If it gets swapped for another slider, I think we should adopt the values from the new slider, rather than override them.